TypeScript error: 'cache' imported from 'react' instead of 'next/cache' in fetchOwner.tsx

## Issue Summary

TypeScript error in `apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx`:

```
app/analyze/(org)/[owner]/fetchOwner.tsx(3,10): error TS2305: Module 'react' has no exported member 'cache'.
```

## Affected File

- `apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx`

## Problem

The file imports `cache` from 'react':

```tsx
import { cache } from 'react';
```

However, `cache` is not exported from the 'react' package in React 18.2.0. In Next.js 13+, `cache` should be imported from 'next/cache'.

## Impact

- TypeScript type checking fails
- Potential runtime issues if the import is not resolved correctly

## Suggested Fix

Change the import from:

```tsx
import { cache } from 'react';
```

To:

```tsx
import { cache } from 'next/cache';
```

## Priority

**Medium** - Type checking fails, but build succeeds (likely due to ignoreBuildErrors configuration)

--- a/apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx
+++ b/apps/web/app/analyze/(org)/[owner]/fetchOwner.tsx
@@ -1,6 +1,6 @@
 import { getOwnerInfo } from '@/components/Analyze/utils';
 import { notFound } from 'next/navigation';
-import { cache } from 'react';
+import { cache } from 'next/cache';
 
 export const fetchOwnerInfo = cache(async (
   orgName: string,

